ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -30,15 +30,39 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-ci-tests-aro-cluster-density-v2
+- as: ocp-qe-perfscale-ci-tests-node-density-cni-6nodes-aro
   cron: 0 12 * * 3
   steps:
     cluster_profile: azure-qe
     env:
+      ARO_MASTER_VM_SIZE: Standard_D8s_v3
       ARO_WORKER_COUNT: "6"
       ARO_WORKER_VM_SIZE: Standard_D8s_v3
     test:
-    - chain: openshift-qe-cluster-density-v2
+    - chain: openshift-qe-node-density-cni
```

Comment:
@vishnuchalla Same comment. Is the intention here not to run cluster-density-v2 tests for ARO?

Reply:
the cluster-density-v2 test is in the chain in line 65